configure

```diff
@@ -1013,6 +1019,9 @@ def configure_openssl(o):
   o['variables']['openssl_no_asm'] = 1 if options.openssl_no_asm else 0
   if options.use_openssl_ca_store:
     o['defines'] += ['NODE_OPENSSL_CERT_STORE']
+  if options.openssl_system_ca_path:
+    o['defines'] += ['NODE_OPENSSL_SYSTEM_CERT_PATH '
+        + str(options.openssl_system_ca_path)]
```

Comment:
`NODE_OPENSSL_SYSTEM_CERT_PATH=...` instead of `NODE_OPENSSL_SYSTEM_CERT_PATH ...`?

Operator should go on the first line.

Isn't `options.openssl_system_ca_path` already a string?

Reply:
Sorry it is, this was a left over from when I use the append action instead of store. I'll fix this.